src/NaadVars.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Bcgov\NaadConnector;
+
+/**
+ * Class NaadVars
+ *
+ * This class is responsible for managing environment variables used in the
+ * application. It retrieves values from the environment or uses default values
+ * when necessary. The class provides a dynamic way to access these variables
+ * through the magic __get and __set methods.
+ *
+ * Environment variables are essential for configuring the application without
+ * hardcoding sensitive information or settings. This class specifically handles
+ * database and application-related environment variables.
+ *
+ * @category Utility
+ * @package  Bcgov\NaadVars
+ * @author   Richard O'Brien <Richard.Obrien@gov.bc.ca>
+ * @license  https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link     https://alerts.pelmorex.com/
+ */
```

Comment:
```suggestion
 * @link     https://alerts.pelmorex.com/
 * @property-read string $naadName The instance of NAAD the application is connecting to, eg. NAAD-1, NAAD-2.
 */
```
If we add these phpdoc comments for each property then they'll show up in intellisense.

Reply:
done